jujugui/static/gui/src/app/components/entity-header/entity-header.js

```diff
@@ -258,11 +258,27 @@ YUI.add('entity-header', function() {
         </li>);
     },
 
+    /**
+      Generates the list of series. Supports bundles, multi-series and
+      single-series charms.
+
+      @method_generateSeriesList
+    */
+    _generateSeriesList: function() {
+      var series = this.props.entityModel.get('series');
+      if (!series) {
+        return null;
+      }
+      if (Array.isArray(series)) {
```

Comment:
To avoid repeating the node code twice, what about:
```javascript
if (!Array.isArray(series)) {
  series  = [series];
}
return series.map(series => 
  <li key={series} className="entity-header__series">{series}</li>);
```